src/Engine/Graphics/Outdoor.cpp

```diff
@@ -2599,7 +2491,8 @@ void ODM_ProcessPartyActions() {
         }
     }
 
-    if (integer_sqrt(pX_ * pX_ + pY_ * pY_ + pZ_ * pZ_) < 8)  // отключить  звук ходьбы при остановке
+    // mute the walking sound when stopping
+    if (pParty->_movementTallySQ < 64)
         pAudioPlayer->StopAll(804);
```

Comment:
This will trigger when _movementTallySQ is still accumulating movement, so likely not what we need. Let's throw in a TODO.